p2p/peer.py

```diff
@@ -812,6 +812,10 @@ def _add_peer(self,
             # This is kept separate from the `expected_exceptions` to be sure that we aren't
             # silencing an error in our authentication code.
             self.logger.info('Got bad auth ack from %r', remote)
+        except MalformedMessage:
+            # This is kept separate from the `expected_exceptions` to be sure that we aren't
+            # silencing an error in our authentication code.
+            self.logger.info('Got malformed response from %r', remote)
```

Comment:
I updated to use `ERROR` for this statement so it stands out more and a second `DEBUG` logging statement that will include the full stack trace so that we can get it out of the debug logs if we find there's an issue here (but so that it doesn't dominate the `INFO` logs).  Are you ok with that approach?

Reply:
Went with the strategy of using `ERROR` for this message  but dumping full stacktrace in `DEBUG` logs.